README.md

```diff
@@ -40,6 +40,19 @@ sizes="320x50,300x100,300x50,468x60,980x250,970x90"
 custom-targeting="key1:value1, key2:value2"]
 ```
 
+##### Custom sizes in shortcode
+
+```
+[ampad ad-unit="<ad-unit-name>"
+sizes=""
+desktop-sizes="320x100,300x100"
+tablet-sizes="468x60,300x100"
+mobile-sizes="300x100,320x50"
+custom-targeting="key1:value1, key2:value2"]
+```
+
+Note: Pass blank string to sizes attributes if you are passing custom sizes attributes.
```

Comment:
@sagarnasit Can we have check instead of passing sizes attribute in case of custom sizes?

Reply:
@pradeep910 I have updated this with check and proper example in the readme.